docs/examples.md

```diff
@@ -63,13 +64,13 @@ import jenkins.automation.builders.BaseJobBuilder
 ```
 import jenkins.automation.builders.FlowJobBuilder
 
-    def oahMaster= new FlowJobBuilder(
+    def flowExampleJob= new FlowJobBuilder(
             name: 'GeneratedFlowJob',
             description: 'this our first stab at it',
             jobs:['job1', 'job2']
```

Comment:
@vickumar1981 added a jobFlow string variable for Flows that include more than 'build(job)' syntax, so we should mention that here.